CHANGELOG.md

```diff
@@ -3,6 +3,33 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.17.2
+
+_Sep 8, 2022_
+
+We'd like to offer a big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.17.2` / `@mui/x-data-grid-pro@v5.17.2` / `@mui/x-data-grid-premium@v5.17.2`
```

Comment:
I assume that's what @LukasTy  is proposing to be consistent with the unmodified community and pro

```suggestion
### `@mui/x-data-grid-premium@v5.17.2`
```

Reply:
Matheus pushed a fix on the Community package so the discussion is closed I guess
But I agree :+1: